src/DynamoPackages/PackageLoader.cs

```diff
@@ -200,9 +202,45 @@ private void TryLoadPackageIntoLibrary(Package package)
             {
                 var dynamoVersion = VersionUtilities.PartialParse(DynamoModel.Version);
 
+                package.EnumerateAdditionalFiles();
+
+                // If the additional files contained an extension manifest, then request it be loaded.
+                var extensions = package.AdditionalFiles.Where(
+                    file => file.Model.Name.Contains("ExtensionDefinition.xml") || file.Model.Name.Contains("ViewExtensionDefinition.xml")).ToList();
+
+                AssemblyLoadContext pkgLoadContext = AssemblyLoadContext.Default;
+
+                bool isolateAnyPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("IsolateAnyPackage", false) ?? false;
+                bool isolateThisPackage = DynamoModel.FeatureFlags?.CheckFeatureFlag("IsolatePackage_" + package.Name, false) ?? false;
+                if (isolateAnyPackage || isolateThisPackage)
+                {
+                    if (extensions.Count() == 1)
+                    {
+                        var ext = extensions[0];
+                        string entryPoint = string.Empty;
+                        try
+                        {
+                            entryPoint = ExtensionLoader.GetExtensionPath(ext.Model.FullName);
+                            if (!string.IsNullOrEmpty(entryPoint))
+                            {
+                                pkgLoadContext = new PkgAssemblyLoadContext(package.Name + "@" + package.VersionName, Path.Combine(package.RootDirectory, entryPoint));
+                            }
+                        }
+                        catch (Exception ex)
+                        {
+                            Log(ex);
+                        }
+                    }
+                    else if (package.Header.node_libraries.Any())
+                    {
+                        string entryPoint = new AssemblyName(package.Header.node_libraries.First()).Name + ".dll";
+                        pkgLoadContext = new PkgAssemblyLoadContext(package.Name + "@" + package.VersionName, Path.Combine(package.BinaryDirectory, entryPoint));
+                    }
+                }
+
```

Comment:
so the logic here should be:
First check if we have any extensions (ext or viewExt) and set the AssemblyLocation from the xml as the ALC resolver directory.
Secondly check if we have node_libs set, and try to use that location as the ALC resolver dir.

Reply:
any other ideas here ?

Reply:
the node libraries should almost always be in the bin folder I would think?

Reply:
does it not work to make the package root folder the plugin location?

Reply:
`make the package root folder the plugin location` that might work.
  I will try that